guides/running-a-discovery-kick-off-workshop.md

```diff
@@ -23,46 +23,74 @@ of the roadmapping questions.
 
 1. **Why are we doing this work?**
 
-   What is the motivation? What are we trying to do or decide? And why now? What
-   is the trigger?
+   What is the motivation? What are we trying to do or decide?
+
+   And why now? What is the trigger?
 
 1. **Who are the users?**
 
-   Are they members of the public, professionals or staff? What different
-   groups, circumstances and behaviours might be important? Are they current or
+   Are they members of the public, professionals or staff? Are they current or
    potential users?
 
+   What different groups, job roles, circumstances, environments, access needs
+   and support needs might be important? Who might currently be worst served and
+   most likely to experience barriers?
+
+   How will we engage and involve the different people we need to work with?
+
 1. **What organisations and stakeholders are involved?**
 
+   Are they colleagues from our organisation or other public sector bodies? What
+   community, civil society, professional and advocacy groups might be
+   important?
+
    What is their relationship to the work? What expectations and concerns might
    they have? How closely might they be involved? How might they help?
 
 1. **What outcomes will users get?**
 
-   What do we know about their intentions? What’s working that we might we build
-   on? What problems might we solve? What unintended consequences might we
-   create?
+   What do we know about their intentions? What problems might we solve for
+   them?
+
+   What’s working that we might we build on? What unintended consequences might
```

Comment:
Typo: 

>  that we might we build

Reply:
Was it meant to say "that we might build on"? There's an extra "we".